workspaces/ui/Taskfile.yml

```diff
@@ -16,18 +16,26 @@ tasks:
 
   demo:docker:build:
     desc: Build and package demo site as a Docker container
-    deps: ["demo:build"]
+    vars:
+      TAG:
+        sh: echo $(../../bin/clean-docker-tag $TAG)
     cmds:
-      - docker build . -f Dockerfile_demo -t $DOCKER_REGISTRY/$REPO:$TAG
+      - task: demo:build
+      - docker build . -f Dockerfile_demo -t $DOCKER_REGISTRY/$REPO:{{.TAG}}
 
   demo:docker:push:
-    - docker push $DOCKER_REGISTRY/$REPO:$TAG
-    - |
-      if [[ "$TAG" =~ ^release-b ]]
-      then
-        docker tag $DOCKER_REGISTRY/$REPO:$TAG $DOCKER_REGISTRY/$REPO:latest
-        docker push $DOCKER_REGISTRY/$REPO:latest
-      fi
+    vars:
+      TAG:
+        sh: echo $(../../bin/clean-docker-tag $TAG)
```

Comment:
the var is set at the task level to not pollute the global scope. invoking any task through the top-level taskfile will load all included taskfiles and add global scope vars/env.